framework/source/class/qx/data/controller/List.js

```diff
@@ -190,6 +190,42 @@ qx.Class.define("qx.data.controller.List",
       event: "changeDelegate",
       init: null,
       nullable: true
+    },
+    
+    /**
+     * Whether a special "null" value is included in the list
+     */
+    allowNull :
+    {
+      apply: "_applyAllowNull",
+      event: "changeAllowNull",
+      init: false,
+      nullable: false,
+      check: "Boolean"
+    },
```

Comment:
Since this adds nullValueTitle and nullValueIcon properties, could allowNull be elided, and references to it assume that a null value is allowed if either (or both) nullValueTitle or nullValueIcon is set?

Reply:
I know what you mean, and I did consider that, but I like the ability to turn this on and off explicitly - icons are almost secondary and IMHO the explicit clarity is preferable.